src/error/mod.rs

```diff
@@ -78,13 +79,18 @@ impl RoverError {
         &self.metadata.suggestion
     }
 
-    pub fn print(&self) -> io::Result<()> {
+    pub fn print(&self, as_markdown: bool) -> io::Result<()> {
         if let Some(RoverClientError::OperationCheckFailure {
             graph_ref: _,
             check_response,
         }) = self.error.downcast_ref::<RoverClientError>()
         {
-            stdoutln!("{}", check_response.get_table())?;
+            let markdown_output_mode = match as_markdown {
+                true => Some(MarkdownOutputMode::Failed),
+                false => None,
+            };
+
+            stdoutln!("{}", check_response.get_table(markdown_output_mode))?;
         }
 
         stderr!("{}", self)?;
```

Comment:
@EverlastingBugstopper I'm not sure what an error could look like at this moment. Do you have an example?

Reply:
great question! check out [this blog post](https://myrrlyn.net/crates/calm_io) for a full writeup from the author of [calm_io](https://crates.io/crates/calm_io) which is where these macros come from.

Reply:
Thank you! My bad I meant what `Self`.fmt could look like at this moment. Should it be included in the md output too?

Reply:
oh i see now, thanks for clarifying.

this will just be the `Display` impl on `RoverError` ([src](https://github.com/apollographql/rover/blob/a8542bea50c0d5f1366b9be57a529ee22cce4cbf/src/error/mod.rs#L120)), so it would look like the errors do in the terminal. since we have those other sections above i don't think we'll need to print that as well. 

thanks for your work on this - i'll have time for a more thorough review next week!